app/Http/Controllers/SchoolController.php

```diff
@@ -0,0 +1,109 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Models\School;
+use Illuminate\Http\Request;
+use Illuminate\Support\Str;
+
+class SchoolController extends Controller
+{
+
+    public function uuid()
+    {
+        return Str::uuid()->toString();
+    }
+    
+    /**
+     * Display a listing of the resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function index()
+    {
+        //
+    }
+
+    /**
+     * Show the form for creating a new resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function create(Request $request)
+    {
+        $id = $this->uuid();
+        $schoolName = $request->get('schoolName');
```

Comment:
Biko where are we getting the school name and school code from? I no sabi Laravel, mind explaining?

Reply:
From the frontend. Post request

Reply:
Ahh  PHP ehnn, I will pass sha